Debug/Core/EcsWorld.cs

```diff
@@ -8,6 +8,10 @@ public sealed partial class EcsWorld
            return (T) GetPool<T>().GetItem(entity);
         }
         
+        internal bool HasItem<T>(int entity) where T : struct, IEcsComponent
```

Comment:
please use `GetEntityComponentsTypeMask(entityId).ContainsType<T>();` for simpler control flow and less code